### DISCUSSION TITLE
Script Troubleshooting Request

### DISCUSSION THREAD

**Discussion by dragonash2019** [kinsi55/CS_BeatSaber_Camera2#30] [Q&A] [answered]
Wondering if you can assist.
I have your sample script working and plugged in my own values with a successful test.
However, when I enter a 4th movement position, I seem to break the script.
When I remove that last position, the script works again. Sop obviously the issue is between the keyboard and the chair! :)

I have a feeling I'm missing a stupid bracket right at the end somewhere, just can't seem to figure it out.

> {
  "syncToSong": false,
  "frames": [
    {
      "position": {
        "x": 1.644,
        "y": 1.766,
        "z": -1.604
      },
      "FOV": 78,
      "rotation": {
        "x": 12.666,
        "y": 337.0768,
        "z": 4.25
      },
      "holdTime": 5
    },
    {
      "position": {
        "x": 2.176,
        "y": 0.357,
        "z": -0.221
      },
      "FOV": 78,
      "duration": 10,
      "holdTime": 5,
      "rotation": {
        "x": 344.554,
        "y": 289.901,
        "z": 3.783
      },
      "transition": "Eased"
    },
    {
      "position": {
        "x": -1.93,
        "y": 2.835,
        "z": 0.057
      },
      "FOV": 78,
      "duration": 10,
      "holdTime": 5,
      "rotation": {
        "x": 36.269,
        "y": 59.942,
        "z": 347.687
    },
	{
      "position": {
        "x": -0.086,
        "y": 2.472,
        "z": 2.56
      },
      "FOV": 78,
      "duration": 10,
      "holdTime": 5,
      "rotation": {
        "x": 29.999,
        "y": 176.939,
        "z": 359.67
	}
    }
  ]
}

**Comment by kinsi55** [comment, marked as answer]
You are missing a closing bracket here on your third frame
![grafik](https://user-images.githubusercontent.com/8348306/117022375-7d5c4d00-acf8-11eb-98c5-f95a33461160.png)

I strongly recommend you to use an Editor that properly supports JSON like Notepad++ and always keep your indentation consistent to make errors like this obvious

**Reply by dragonash2019** [reply]
oic. I thought that bracket below it was the correct one. Sorry for the noob question. New to this, but thanks again!
And thanks for the notepad++ tip

It's working now :)
